src/components/rocket.jsx

```diff
@@ -0,0 +1,39 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { fetchRockets, reserveRocket } from '../slices/featureRocket/rocketSlice';
+import styles from '../styles/rocket.css';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const { rockets } = useSelector((state) => state.rockets);
+  const { status } = useSelector((state) => state.rockets);
+
+  useEffect(() => {
+    if (status === false) dispatch(fetchRockets());
+  }, [dispatch, status]);
+
+  return (
+    <div className="container">
+      {status && rockets.map((each) => (
+        <article key={each.id} className="article">
+          <img src={each.image} className="img" alt={each.name} />
+          <div>
+            <h2 className={styles.title}>{each.name}</h2>
+            <p className="rocketDetails">
+              {each.reserved ? <span className="span">Reserved</span> : ''}
+              {each.description}
+            </p>
+            <button
+              type="button"
+              className={each.reserved ? styles.unreserve : styles.reserve}
```

Comment:
While you did an amazing job working on the project requirements you missed out some things. It would be nice if you followed the wire frame design.

This is your design:
![Screenshot 2023-04-14 at 3 36 58 PM](https://user-images.githubusercontent.com/79506105/232059167-1dcb1278-8702-45bb-aa00-625e410a69a1.png)


Please stick to this design:



![Home page ](https://user-images.githubusercontent.com/79506105/232059322-70fb3972-1124-43a4-9a03-8a15d082ba92.png)

Please pay attention to detail. Follow design guideline make sure all fonts, borders and colors are added as per design/wireframe rules.